specification/metrics/semantic_conventions/hardware-metrics.md

```diff
@@ -270,7 +270,7 @@ fiber channel port or a Wi-Fi adapter.
 | `hw.errors`                        | Number of errors encountered by the network adapter                                                          | {error}  | Counter                                           | Int64      | `hw.error.type` (Recommended) | `zero_buffer_credit`, `crc`, etc. |
 |                                    |                                                                                                              |          |                                                   |            | `hw.type` (**Required**)      | `network`                         |
 |                                    |                                                                                                              |          |                                                   |            | `direction` (Recommended)     | `receive`, `transmit`             |
-| `hw.network.bandwidth.limit`       | Link speed                                                                                                   | By       | UpDownCounter                                     | Int64      |                               |                                   |
+| `hw.network.bandwidth.limit`       | Link speed                                                                                                   | By/s     | UpDownCounter                                     | Int64      |                               |                                   |
```

Comment:
I agree with you it's odd to think of the link speed as a UpDownCounter that can be incremented/decremented. The operational link speed is whatever is negotiated by both ends of the network link.

The link speeds can be added across multiple ports in a network device. For example, a switch may have 32 physical ports, 16 of them are configured, and at any time the operational state may change depending on the link status.

Reply:
The use of `UpDownCounter`  for `*.limit` metrics has been discussed in this [this PR](https://github.com/open-telemetry/opentelemetry-specification/pull/2518). I understand it's a bit counter-intuitive (I was fighting against this myself until I understood what it really meant 😅). The key is: It's a summable metric (calculating the sum of bandwidth for all network interfaces of a system is a valid use case, unlike summing up bandwidth utilization values). That's why `UpDownCounter` is the proper metric type for `hw.network.bandwidth.limit`.

Also, the available bandwidth of a network interface can change over time depending on the link negotiation. If you monitor your Wifi adapter, you'll see its speed changing constantly, sometimes several times per minute. This is one more reason for using `UpDownCounter`.

Reply:
Perhaps then we need to better document these to clarify any future confusion? When we convert the markdown to YAML to generate these, then we can use `note`, but for now it can be just a foot note or something. What do you think?

Reply:
BTW, the "link speed" is ambiguous. It could be the configured link speed or the operational link speed. Some network interfaces have both a concept of configured and operational link speed.

Reply:
Just my 2 cents, opening a separate PR seems a better option, otherwise this simple fix may end up being delayed.

Reply:
How does one determine whether a metric should be considered additive or not? In most cases, it's obvious. But what about `hw.power` which is a gauge. Yes, it does not make sense to sum `hw.power` measurements taken at different timestamps, but it may be useful to sum `hw.power` measurements from multiple devices at a given timestamp. For example, that could be used to analyze the power drawn versus the circuit capacity.

Reply:
@sebastien-rosset  

> How does one determine whether a metric should be considered additive or not

The spec has some guidelines https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/supplementary-guidelines.md#guidelines-for-instrumentation-library-authors

> may be useful to sum hw.power measurements from multiple devices at a given timestamp
You can maybe achieve this by summing them up in the backend? I didn't read them in full so I might be off here.